src/common/components/templates/InfoBanner.tsx

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import { useState, useEffect } from "react";
+import { useRouter } from "next/router";
+import { FaTimes } from "react-icons/fa";
+import * as Toast from "@radix-ui/react-toast";
+
+export default function InfoToast() {
+  const [isDisplayed, setIsDisplayed] = useState(false);
+  const [message, setMessage] = useState("");
+  const [duration, setDuration] = useState(10000);
+  const router = useRouter();
+  const { pathname, query } = router;
+  const userFarcasterName = query.handle;
+
+  useEffect(() => {
+    let storedState;
+
+    if (pathname === "/homebase") {
+      storedState = localStorage.getItem("homebaseToastDisplayed");
+      if (!storedState && localStorage.getItem("privy:token") !== null) {
+        setIsDisplayed(true);
+        setMessage(
+          "Your homebase is a space that only you can see. Click the paintbrush to customize it 🚀",
+        );
+        setDuration(10000);
+      } else {
+        setIsDisplayed(false);
+      }
+    } else if (
+      pathname.startsWith("/s/") &&
+      userFarcasterName === "farcaster"
+    ) {
+      storedState = localStorage.getItem("profileToastDisplayed");
+      if (!storedState) {
+        setIsDisplayed(true);
+        setMessage(
+          "This is your profile. Click the paintbrush to customize your space.",
+        );
+        setDuration(10000);
+      } else {
+        setIsDisplayed(false);
+      }
+    } else {
+      setIsDisplayed(false);
+    }
+  }, [pathname, userFarcasterName]);
+
+  const closeToast = () => {
+    setIsDisplayed(false);
+    if (pathname === "/homebase") {
+      localStorage.setItem("homebaseToastDisplayed", "true");
+    } else if (pathname.startsWith("/s/")) {
+      localStorage.setItem("profileToastDisplayed", "true");
+    }
+  };
+
+  return (
+    <Toast.Provider>
+      <Toast.Root
+        open={isDisplayed}
+        onOpenChange={setIsDisplayed}
+        duration={duration} // Set the duration here
+        className="fixed bottom-4 right-4 p-4 bg-blue-100 border border-blue-300 rounded-md shadow-lg"
+      >
+        <div className="flex items-center justify-between">
+          <div className="flex items-center">
+            <img
+              src="https://i.ibb.co/L8Fb37T/image.png"
```

Comment:
Please move this to be a constant variable defined at the start of the file. That way it will be easier to locate and update if we ever need to change it and it offers a single source of truth

Reply:
ok, i left the hardcoded image there cause i didnt know if the infoicon would win over toms head. Just talked to tom over the phone and he said its ok using his image for the toast so I will add it as a constant at the start of the file!

Reply:
I am adding this image to /public/imagess , cool ?